### PR TITLE
Changing the default field function for uid

### PR DESCRIPTION
**Pull request by bluemoon**
@charles-vdulac

### CODE REVIEW

**Comment by charles-vdulac** [comment]
We need a migration file.

But `makemigrations` generates a `ValueError: Cannot serialize`.
To solve it, replace `uid = models.CharField` by 

```
from shortuuidfield import ShortUUIDField
uid = ShortUUIDField(...)
```

Cf https://xxx-cook-book.gitbooks.io/django-cook-book/content/Databases/Migrations.html

Note: to generate the migration, run `python manage.py makemigrations` from a lib which uses this lib.

**Comment by bluemoon** [comment]
@charles-vdulac this is good to go now

File: setup.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
         'django-oauth-toolkit>=0.9.0',
         'python-social-auth>=0.2.2',
         'shortuuid>=0.4.3',
+        'django-shortuuidfield==0.1.3',
```
Comment (charles-vdulac) [inline setup.py]: As `locarise_drf_oauth2_support` is a third dependency, it's better to let the main project to install a newer version of `django-shortuuidfield` with `'django-shortuuidfield>=0.1.3'`

Comment (bluemoon) [inline setup.py]: 👍

**Pull request merged**